tests/distributed/test_pynccl.py

```diff
@@ -14,7 +14,9 @@ def distributed_run(fn, world_size):
     for i in range(number_of_processes):
         env = os.environ.copy()
         env['RANK'] = str(i)
+        env['LOCAL_RANK'] = str(i)
         env['WORLD_SIZE'] = str(number_of_processes)
+        env['LOCAL_WORLD_SIZE'] = str(number_of_processes)
```

Comment:
Should we also test the case where `LOCAL_*` is not present in env?

Reply:
The tests using `ray` already tested this, i.e. the default value of `local_rank=-1`.

Reply:
got it, thanks!